pkg/result/processors/invalid_issue.go

```diff
@@ -9,6 +9,9 @@ import (
 
 var _ Processor = (*InvalidIssue)(nil)
 
+// InvalidIssue filters invalid reports.
+// - non-go files (except `go.mod`)
+// - reports without file path
```

Comment:
Please fix the list formatting:

<img width="795" alt="image" src="https://github.com/user-attachments/assets/3a6a6a84-1f59-4853-aaed-75a16fc47e09" />